rest-api/alembic/versions/0e92151ebd4a_amend_biobank_orders.py

```diff
@@ -48,6 +48,7 @@ def upgrade_rdr():
     op.add_column('biobank_order', sa.Column('restored_site_id', sa.Integer(), nullable=True))
     op.add_column('biobank_order', sa.Column('restored_time', model.utils.UTCDateTime(), nullable=True))
     op.add_column('biobank_order', sa.Column('restored_username', sa.String(length=255), nullable=True))
+    op.add_column('biobank_order', sa.Column('version', sa.Integer(), nullable=False))
```

Comment:
So since all of our existing DBs should probably have run through these migrations, the change you're doing here -- deleting one migration and merging it into here -- is *probably* safe. But I wouldn't recommend it. If you don't have any schema changes to make, I'd just revert all the Alembic changes in this PR.